src/components/study/Registration.vue

```diff
@@ -29,37 +29,36 @@
 import MainNav from './MainNav.vue'
 import store from '../store'
 import axios from 'axios';
-import { VueTelInput } from 'vue-tel-input'
+import VuePhoneNumberInput from 'vue-phone-number-input'
 
 export default {
   name: 'StudyRegistration',
-  components: { MainNav, VueTelInput },
+  components: { MainNav, VuePhoneNumberInput },
   data() {
     return {
       phoneValue: '',
       countryCodes: ['CA','US','NL'],
       defaultCountry: this.$t('registration-screen.default-region-code'),
       selectedRegionCode: '',
+      isValid: false,
     }
   },
   computed: {
-    hasNumber: function() {
-      return ( this.phoneValue && this.phoneValue.replace(/\D/g,'').length >= 10 ) ? 1 : .5
-    },
   },
   methods: {
-    onCountryChange: function (countryObject) {
-      this.selectedRegionCode = countryObject.iso2
+    onUpdate: function (updatedObject) {
+      this.selectedRegionCode = updatedObject.countryCode
+      this.isValid = updatedObject.isValid
```

Comment:
Their update event sends us both the countryCode and isValid (so I removed our hasNumber client-side validation in favor of theirs)